src/SDKs/ApiManagement/Management.ApiManagement/generate.ps1

```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "apimanagement/resource-manager" -AutoRestVersion "latest"
\ No newline at end of file
+Start-AutoRestCodeGeneration -ResourceProvider "apimanagement/resource-manager" -AutoRestVersion "latest" -AutoRestCodeGenerationFlags "--tag=package-2018-01"
```

Comment:
Please set this tag as the default tag in the REST spec. Is there a reason why you are picking a particular tag here?

Reply:
Because we want the documentation to be generated off the -preview version, but the SDK to be targeted for specific stable version.

Reply:
IIRC documentation is generated from the published nuget package, so SDK and documentation should ideally be the same version
@shahabhijeet could you please take a look